appyters/dimensionality_reduction_visualization/dimensionality_reduction_visualization.ipynb

```diff
@@ -387,12 +490,12 @@
     "    caption = \"t-SNE 2D scatter plot. Each point presents a column from the input data matrix.\"\n",
     "\n",
     "diplay_caption(counter, caption)\n",
+    "counter += 1\n",
```

Comment:
A fun way to make this slightly more pythonic is to use

```python
# at the top
import itertools
counter = iter(itertools.count(start=1))


# everywhere else
display_caption(next(counter), caption)
```

Reply:
Great idea! Just changed it.